themes/default/content/blog/google-cloud-pulumi-import-account-scraper/index.md

```diff
@@ -0,0 +1,261 @@
+[The full code](https://github.com/pulumi/pulumi-import-google-cloud-account-scraper) also contains some additional mapping and processing for Google Cloud IAM resources, but it follows the same basic patterns as the mapping for other types.
+
+## Importing to Pulumi
+
+Now that we have our conversion script written, we can run it with the following command:
+
+```bash
+python3 main.py -i sample-output.yaml -o pulumi-import.json -p <your-project-id>
+```
+
+The command will generate output like the following:
+
+```json
+{
+  "resources": [
+    {
+      "type": "gcp:artifactregistry/repository:Repository",
+      "name": "gcf-artifacts",
+      "id": "europe-west1/gcf-artifacts"
+    },
+    {
+      "type": "gcp:compute/disk:Disk",
+      "name": "gke-helloworld-2a71d4f-pvc-1e6aa931-d742-437e-a52b-413999ace2be",
+      "id": "us-central1-a/gke-helloworld-2a71d4f-pvc-1e6aa931-d742-437e-a52b-413999ace2be"
+    },
+// ...
+```
+
+Once we're in our Pulumi program's root, we can instruct `pulumi import` to do a bulk import and output the generated code to a file in the language of our program (this example includes TypeScript output, but `pulumi import` will automatically generate code in the correct language for your project).
+
+```bash
+pulumi import -f ../config-connector-transform/pulumi-import.json -y -s dev -o index.ts
+```
+
+And we can see that in our Pulumi program, all the attributes of our imported resources are present, which means no further actions are (typically - see note below for known exceptions) necessary! We are ready to manage our critical infrastructure as code from now on!
+
+```typescript
+const gcf_artifacts = new gcp.artifactregistry.Repository("gcf-artifacts", {
+    description: "This repository is created and used by Cloud Functions for storing function docker images.",
+    format: "DOCKER",
+    labels: {
+        "goog-managed-by": "cloudfunctions",
+    },
+    location: "europe-west1",
+    project: "your-project-id",
+    repositoryId: "gcf-artifacts",
+});
+
+const gke_helloworld_2a71d4f_pvc_1e6aa931_d742_437e_a52b_413999ace2be = new gcp.compute.Disk("gke-helloworld-2a71d4f-pvc-1e6aa931-d742-437e-a52b-413999ace2be", {
+    description: "{\"kubernetes.io/created-for/pv/name\":\"pvc-1e6aa931-d742-437e-a52b-413999ace2be\",\"kubernetes.io/created-for/pvc/name\":\"wpdev-wordpress\",\"kubernetes.io/created-for/pvc/namespace\":\"default\"}",
+    labels: {
+        "goog-gke-volume": "",
+    },
+    name: "gke-helloworld-2a71d4f-pvc-1e6aa931-d742-437e-a52b-413999ace2be",
+    physicalBlockSizeBytes: 4096,
+    project: "your-project-id",
+    size: 10,
+    zone: "us-central1-a",
+});
+```
+
+{{% notes %}}
+You may need to massage some of the generated output for any `*IAMPolicy` resources because deleted principals cannot be contained in IAM policies due to a limitation of the Google Classic provider, but they still contained in the generated code returned by `pulumi import`, which in turn comes from the Google API.
```

Comment:
```suggestion
You may need to massage some of the generated output for any `*IAMPolicy` resources because deleted principals cannot be contained in IAM policies due to a limitation of the Google Classic provider, but they are still contained in the generated code returned by `pulumi import`, which in turn comes from the Google API.
```